index.js

```diff
@@ -81,6 +82,49 @@ rbush3d.prototype = {
         return false;
     },
 
+    raycastInv: function (ox, oy, oz, idx, idy, idz, length) {
+        length = length || Infinity;
```

Comment:
Avoid assignment to arguments.  This may cause a deoptimization.

Reply:
I found `tsc` use same way to handle default argument , so is there a way to set a default argument and void assignment?
```javascript
// function test(a:number, b = 666)
function test(a, b) {
    if (b === void 0) { b = 666; }
    return a + b;
}
```
Or how about port the code to typescript to avoid handle the thing?